perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/UsersManagerMethod.java

```diff
@@ -507,12 +507,29 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * @throw RelationExistsException When the User has some members assigned.
 	 * @throw AnonymizationNotSupportedException When an attribute should be anonymized but its module doesn't specify the anonymization process.
 	 */
+	/*#
+	 * Anonymizes user (force) - according to configuration, each of user's attributes is either
+	 * anonymized, kept untouched or deleted. Also deletes other user's related data, e.g.
+	 * authorships of users publications, mail change and password reset requests, bans...
+	 * Also removes associated members.
+	 *
+	 * @param user int User <code>id</code>
+	 * @param force boolean If true, use force anonymization
+	 * @throw UserNotExistsException When the User specified by <code>id</code> doesn't exist.
+	 * @throw RelationExistsException When the User has some members assigned.
+	 * @throw AnonymizationNotSupportedException When an attribute should be anonymized but its module doesn't specify the anonymization process.
+	 */
 	anonymizeUser {
 
 		@Override
 		public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			parms.stateChangingCheck();
-			ac.getUsersManager().anonymizeUser(ac.getSession(), ac.getUserById(parms.readInt("user")));
+
+			if (parms.contains("force") && parms.readBoolean("force")) {
+				ac.getUsersManager().anonymizeUser(ac.getSession(), ac.getUserById(parms.readInt("user")), true);
+			} else {
+				ac.getUsersManager().anonymizeUser(ac.getSession(), ac.getUserById(parms.readInt("user")));
```

Comment:
```suggestion
				ac.getUsersManager().anonymizeUser(ac.getSession(), ac.getUserById(parms.readInt("user")), false);
```
Maybe we can do this and keep only the version of anonymizeUser with the force parameter for cleaner code, what do you think?